tests/phpunit/integration/Core/Storage/Post_MetaTest.php

```diff
@@ -0,0 +1,110 @@
+<?php
+/**
+ * Post_MetaTest
+ *
+ * @package   Google\Site_Kit\Tests\Core\Storage
+ * @copyright 2021 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Tests\Core\Storage;
+
+use Google\Site_Kit\Core\Storage\Post_Meta;
+use Google\Site_Kit\Tests\TestCase;
+
+/**
+ * @group Storage
+ */
+class Post_MetaTest extends TestCase {
+
+	public function test_get() {
+		global $wpdb;
+
+		$post_meta  = new Post_Meta();
+		$post_id    = $this->factory()->post->create();
+		$meta_key   = 'test_meta';
+		$meta_value = 'test_value';
+
+		$wpdb->insert(
+			$wpdb->postmeta,
+			array(
+				'post_id'    => $post_id,
+				'meta_key'   => $meta_key,
+				'meta_value' => $meta_value,
+			)
+		);
+
+		$this->assertEquals(
+			$meta_value,
+			$post_meta->get( $post_id, $meta_key, true )
+		);
+	}
+
+	public function test_add() {
+		global $wpdb;
+
+		$post_meta  = new Post_Meta();
+		$post_id    = $this->factory()->post->create();
+		$meta_key   = 'test_meta';
+		$meta_value = 'test_value';
+
+		$meta_id   = $post_meta->add( $post_id, $meta_key, $meta_value );
+		$new_value = $wpdb->get_var( $wpdb->prepare( "SELECT meta_value FROM {$wpdb->postmeta} WHERE post_id = %d AND meta_key = %s", $post_id, $meta_key ) );
```

Comment:
How about we extract the query to the test case like we have for `TestCase::queryOption` ?

Reply:
Updated.